graspologic/plot/plot.py

```diff
@@ -291,6 +286,11 @@ def heatmap(
         msg = "cbar must be a bool, not {}.".format(type(center))
         raise TypeError(msg)
 
+    # Warning on labels
+    if (inner_hier_labels is None) and (outer_hier_labels is not None):
+        msg = "outer_hier_labels requires inner_hier_labels to be used."
+        warnings.warn(msg)
+
```

Comment:
Added this because I tested adding outer_hier_labels without inner_hier_labels, and nothing happened, so I figured users should get a warning in this case.